app/assets/stylesheets/earthworks.css

```diff
@@ -103,4 +103,8 @@
 .sort-pagination, .pagination-search-widgets {
   padding-bottom: .5rem;
   border-bottom: 2px solid var(--bs-border-color);
+}
+
+#main-container .leaflet-container {
+  height: 620px;
```

Comment:
is it still this big on mobile? seems pretty big

Reply:
It looks like on mobile we are supposed to hide the map.

Reply:
Updated the css to include mobile